OpenOversight/test_data.py

```diff
@@ -50,7 +51,7 @@ def populate():
     image2 = models.Image(filepath='static/images/test_cop2.png')
     image3 = models.Image(filepath='static/images/test_cop3.png')
     image4 = models.Image(filepath='static/images/test_cop4.png')
-
+    image5 = models.Image(filepath='static/images/test_cop5.jpg')
     test_images = [image1, image2, image3, image4]
```

Comment:
You need to add this image to `test_images`: 

`test_images = [image1, image2, image3, image4, image5]`